### ISSUE TITLE
Error : Class 'Symfony\Component\Security\Http\Authenticator\AbstractAuthenticator' not found

### ISSUE THREAD

**Issue by debesha** [lexik/LexikJWTAuthenticationBundle#980]
I've got Symfony 4.4 and installed LexikJWTAuthenticationBundle (composer installed a version v2.14.4

However, I'm getting the following error:
```
Error : Class 'Symfony\Component\Security\Http\Authenticator\AbstractAuthenticator' not found
 application/vendor/lexik/jwt-authentication-bundle/Security/Authenticator/JWTAuthenticator.php:39
```

And indeed, Symfony Security Component 4.4 does not have Http\Authenticator\AbstractAuthenticator

**Comment by fd6130** [comment]
Try switch to v2.11.3?

**Comment by debesha** [comment]
Yes, that worked. But may be makes sense to fix composer definition? Obviously, v2.14 doesn't support Symfony 4.4

**Comment by chalasr** [comment]
It does but with a different configuration, as documented.
